src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -124,14 +178,17 @@
 
     render() {
         const Dialogs = this.context;
-        const { idPrefix } = this.props;
-        const { name, description, submitted } = this.state;
+        const { idPrefix, isExternal, vm } = this.props;
+        const { name, description, memoryPath, submitted } = this.state;
         const validationError = this.onValidate(submitted);
 
         const body = (
             <Form onSubmit={e => e.preventDefault()} isHorizontal>
-                <NameRow name={name} validationError={validationError} onValueChanged={this.onValueChanged} />
+                <NameRow name={name} validationError={validationError.name} onValueChanged={this.onValueChanged} />
                 <DescriptionRow description={description} onValueChanged={this.onValueChanged} />
+                {isExternal && vm.state === 'running' &&
+                    <MemoryPathRow memoryPath={memoryPath} onValueChanged={this.onValueChanged}
+                                   validationError={validationError.memory} />}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20240126-150137-d0fc7d9f-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)